pom.xml

```diff
@@ -67,6 +67,13 @@
         </repository>
     </repositories>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <id>repo.jenkins-ci.org</id>
+            <url>http://repo.jenkins-ci.org/public/</url>
```

Comment:
See example in https://github.com/jenkinsci/pom/blob/master/pom.xml#L89L100

Better use `https`.

Fine apart from that.